.circleci/config.yml

```
@@ -36,12 +36,22 @@ jobs:
     steps:
       - trivy-scan:
           image_name: certgenerator
+  functional-test:
+    machine:
+      image: ubuntu-2004:202104-01
+      resource_class: xlarge
```

Do you think we could get by with `large` ?
```suggestion
      resource_class: large
```

---

FYI I tested this and large does work. https://app.circleci.com/pipelines/github/astronomer/certgenerator/12/workflows/46d1383d-6615-480f-ab75-7c6f62f14a27